src/main/java/uk/gov/laa/ccms/caab/bean/validators/costs/CostDetailsValidator.java

```diff
@@ -4,6 +4,7 @@
 import org.springframework.stereotype.Component;
 import org.springframework.validation.Errors;
 import uk.gov.laa.ccms.caab.bean.costs.CostsFormData;
+import uk.gov.laa.ccms.caab.bean.priorauthority.PriorAuthorityFormDataDetails;
```

Comment:
This is the only change in this file, so presume it's unused.

Reply:
amended